packages/flutter/test/widgets/platform_menu_bar_test.dart

```diff
@@ -53,6 +53,7 @@ void main() {
             child: PlatformMenuBar(
               menus: createTestMenus(
                 onActivate: onActivate,
+                onActivateIntent: (String item) => TestPlatformMenuBarIntent(item),
```

Comment:
This should be split into two tests, one that tests `onActivate`, and one that tests `onActivateIntent`, since they can't both be specified at the same time.